### ISSUE TITLE
Add SourceCred Doc

### ISSUE THREAD

**Issue by marvino3** [smartcontractresearchforum/discovery#232]

**Comment by vsterling** [comment]
@ralphlombreglia here's the [SourceCred doc](https://docs.google.com/document/d/17fT7ODvRUy6vAI1srtjL4mlSpzL0Lv-UzFIfYFiaa4Q/edit?usp=sharing) ready for editing

**Comment by ralphlombreglia** [comment]
@vsterling Edited by me yesterday, Wednesday, August 17, 2022

**Comment by isss111** [comment]
@zubepaul Hey Paul! Ralph edited this document and left a few comments in the document. We are mainly missing links to presentations. Can you please add them? Thank you.

**Comment by isss111** [comment]
@zubepaul would you mind directing me to the folders with following presentations:
SourceCred - Node and Edge Weights
SCRF SourceCred Community Vote
SCRF - SourceCred Poll Results and What's Next

Thank you!

**Comment by zubepaul** [comment]
There is a pinned issue in the Engagement Repo that directs people to this documentation. It should have everything you are looking for @isss111 
https://github.com/smartcontractresearchforum/docs/blob/main/en/scrf_scred.md

**Comment by isss111** [comment]
@marvino3 there is already a github version of this doc:
https://github.com/smartcontractresearchforum/docs/blob/main/en/scrf_scred.md

What should we do with this?

**Comment by marvino3** [comment]
Yes, however, this published doc doesn't follow a github doc template and isn't reflected within github docs. I think we want to give it a discovery treatment and then add it to docs.

**Comment by SirAh0** [comment]
Good idea! This document was added very rapidly and then not touched again. The only thing I've been doing is maintaining the monthly tables of cred % scores. 

Happy to help in any way I can just let me know!

**Comment by isss111** [comment]
@marvino3 let me know what you think:
https://hackmd.io/@scrf/rJBxCusUs

@SirAh0 I didn't include the monthly tables of cred % scores because I don't have them.
